Test/Unit/Gateway/Request/AuthorizeRequestTest.php

```diff
@@ -48,14 +61,22 @@ protected function setUp(): void
         $this->order = $this->getMockBuilder(OrderAdapterInterface::class)
             ->disableOriginalConstructor()
             ->getMock();
-        $this->authorizationRequest = new AuthorizationRequest($this->payment);
+        $this->helper = $this->getMockBuilder(Data::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->config = $this->getMockBuilder(Config::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+        $this->authorizationRequest = new AuthorizationRequest($this->helper, $this->config);
+        // $this->authorizationRequest = new AuthorizationRequest($this->payment);
```

Comment:
Can this comment be removed? Is there a reason to keep it?

Reply:
Yes, This comment code is removed now, and commit the changes.